spec/controllers/wishlists_controller_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe WishlistsController do
   context "when user is logged in" do
     before(:each) { sign_in_user }
+    let!(:wishlist) { create(:wishlist, user: controller.current_user) }
```

Comment:
It looks like I've simply followed Your approach (delete action). But of course it isn't an excuse and both "!" and duplication in 'destroy' aren't necessary. Do You think defining wishlist separately in 'delete', 'update' and 'get' is a good option? I prefer to put it here.